e2e-tests/data-at-rest-encryption/compare/statefulset_some-name-cfg.yml

```diff
@@ -146,6 +146,8 @@ spec:
               value: cfg
             - name: PBM_MONGODB_PORT
               value: "27017"
+            - name: PBM_AGENT_SIDECAR
+              value: "true"
             - name: SHARDED
```

Comment:
We have `PMM_AGENT_SIDECAR_SLEEP` option now (time to wait before restarting pmm-agent if PMM_AGENT_SIDECAR is true. 1 second by default. ) @egegunes what do you think about the default value? We may need to use e.g. 5 seconds instead of 1.

Reply:
using 5 seconds makes more sense, but just to be sure we are talking about PBM right?

Reply:
yes, I have corrected the names of vars

Reply:
@pooknull we can remove all these lines as well
 https://github.com/percona/percona-server-mongodb-operator/blob/main/build/pbm-entry.sh#L17-L60